ci/azure/posix.yml

```diff
@@ -9,57 +9,57 @@ jobs:
   strategy:
     matrix:
       ${{ if eq(parameters.name, 'macOS') }}:
-        py36_macos:
-          ENV_FILE: ci/deps/azure-macos-36.yaml
-          CONDA_PY: "36"
+        py37_macos:
+          ENV_FILE: ci/deps/azure-macos-37.yaml
+          CONDA_PY: "37"
           PATTERN: "not slow and not network"
 
       ${{ if eq(parameters.name, 'Linux') }}:
-        py36_minimum_versions:
-          ENV_FILE: ci/deps/azure-36-minimum_versions.yaml
-          CONDA_PY: "36"
+        py37_minimum_versions:
+          ENV_FILE: ci/deps/azure-37-minimum_versions.yaml
+          CONDA_PY: "37"
           PATTERN: "not slow and not network and not clipboard"
 
-        py36_locale_slow_old_np:
-          ENV_FILE: ci/deps/azure-36-locale_slow.yaml
-          CONDA_PY: "36"
+        py37_locale_slow:
+          ENV_FILE: ci/deps/azure-37-locale_slow.yaml
+          CONDA_PY: "37"
           PATTERN: "slow"
           # pandas does not use the language (zh_CN), but should support different encodings (utf8)
           # we should test with encodings different than utf8, but doesn't seem like Ubuntu supports any
           LANG: "zh_CN.utf8"
           LC_ALL: "zh_CN.utf8"
           EXTRA_APT: "language-pack-zh-hans"
 
-        py36_slow:
-          ENV_FILE: ci/deps/azure-36-slow.yaml
-          CONDA_PY: "36"
+        py37_slow:
+          ENV_FILE: ci/deps/azure-37-slow.yaml
+          CONDA_PY: "37"
           PATTERN: "slow"
 
-        py36_locale:
-          ENV_FILE: ci/deps/azure-36-locale.yaml
-          CONDA_PY: "36"
-          PATTERN: "not slow and not network"
-          LANG: "it_IT.utf8"
-          LC_ALL: "it_IT.utf8"
-          EXTRA_APT: "language-pack-it xsel"
+#        py37_locale:
```

Comment:
is there a reason these are commented?

Reply:
Travis has a py37_locale build. So I commented this out. I uncommented it in the last commit. But the CI test timed out. It looks similar to the issue mentioned in #35309.